tests/web_api/test_tracking.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
```

Comment:
Unless this test _actually_ test that the running time is lower than a defined upper bound, this should not be executed each time we run the test.